deploy/bases/app/jobs/install.yaml

```diff
@@ -11,23 +11,43 @@ spec:
         app: magento-install
         k8s-app: magento
     spec:
+      initContainers:
+      - name: wait-for-db
+        image: kiweeteam/magento2:2.3.7-p2-1
```

Comment:
why is the image still at 2.3.7? or this is going to be a first step for the upgrade?

Reply:
I'd recommend building the image for 2.4.3, or drop the version matching with magento version if they don't need to be paired.